draft-ietf-jsonpath-base.md

```diff
@@ -1670,7 +1692,11 @@ instance of `ValueType`.
 | `$[?match(@.timezone, 'Europe/.*') == true]` | not well-typed as `LogicalType` may not be used in comparisons |
 | `$[?value(@..color) == "red"]` | well-typed |
 | `$[?value(@..color)]` | not well-typed as `ValueType` may not be used in a test expression |
-| `$[?bar(1==1)]` | not well-typed, where `bar` is a function with a parameter of type `LogicalType` and result type `LogicalType`, as `1==1` is neither a query nor a function expression with a suitable result type
+| `$[?bar(@.a)]`  | well-typed for any function `bar` with a parameter of any declared type and result type `LogicalType`               |
```

Comment:
Just this "any declared type" to fix. Otherwise I think this PR it's good.

Reply:
Done in 28bb068